pkg/image/importer/importer.go

```diff
@@ -110,10 +110,18 @@ func (i *ImageStreamImporter) importImages(ctx gocontext.Context, retriever Repo
 		if from.Kind != "DockerImage" {
 			continue
 		}
-		ref, err := api.ParseDockerImageReference(from.Name)
-		if err != nil {
-			isi.Status.Images[i].Status = invalidStatus("", field.Invalid(field.NewPath("from", "name"), from.Name, fmt.Sprintf("invalid name: %v", err)))
-			continue
+		var (
+			err error
+			ref api.DockerImageReference
+		)
+		if from.Name != "*" {
```

Comment:
@smarterclayton I already tried this https://gist.github.com/legionus/13042d3682fbba1ca6db53b7c5b1e653 and got:
```
$ oc new-app -L 
error: can't look up Docker image "*":
```
with my changes this command returns:
```
Templates (oc new-app --template=<template>)
-----
cakephp-mysql-example
  Project: openshift
  An example CakePHP application with a MySQL database
dancer-mysql-example
  Project: openshift
  An example Dancer application with a MySQL database
django-psql-example
  Project: openshift
  An example Django application with a PostgreSQL database
jenkins-ephemeral
  Project: openshift
  Jenkins service, without persistent storage.
WARNING: Any data stored will be lost upon pod destruction. Only use this template for testing
jenkins-persistent
  Project: openshift
  Jenkins service, with persistent storage.
You must have persistent volumes available in your cluster to use this template.
jenkins-pipeline-example
  Project: openshift
  This example showcases the new Jenkins Pipeline integration in OpenShift, which performs continuous integration and deployment right on the platform. The template contains a Jenkinsfile - a definition of a multi-stage CI process - that leverages the underlying OpenShift platform for dynamic and scalable builds. OpenShift integrates the status of your pipeline builds into the web console allowing you to see your entire application lifecycle in a single view.
mariadb-ephemeral
  Project: openshift
  MariaDB database service, without persistent storage. WARNING: Any data stored will be lost upon pod destruction. Only use this template for testing
mongodb-ephemeral
  Project: openshift
  MongoDB database service, without persistent storage. WARNING: Any data stored will be lost upon pod destruction. Only use this template for testing
mysql-ephemeral
  Project: openshift
  MySQL database service, without persistent storage. WARNING: Any data stored will be lost upon pod destruction. Only use this template for testing
nodejs-mongodb-example
  Project: openshift
  An example Node.js application with a MongoDB database
postgresql-ephemeral
  Project: openshift
  PostgreSQL database service, without persistent storage. WARNING: Any data stored will be lost upon pod destruction. Only use this template for testing
rails-postgresql-example
  Project: openshift
  An example Rails application with a PostgreSQL database

Image streams (oc new-app --image-stream=<image-stream> [--code=<source>])
-----
jenkins
  Project: openshift
  Tags:    1, latest
mariadb
  Project: openshift
  Tags:    10.1, latest
mongodb
  Project: openshift
  Tags:    2.4, 2.6, 3.2, latest
mysql
  Project: openshift
  Tags:    5.5, 5.6, latest
nodejs
  Project: openshift
  Tags:    0.10, 4, latest
perl
  Project: openshift
  Tags:    5.16, 5.20, latest
php
  Project: openshift
  Tags:    5.5, 5.6, latest
postgresql
  Project: openshift
  Tags:    9.2, 9.4, 9.5, latest
python
  Project: openshift
  Tags:    2.7, 3.3, 3.4, 3.5, latest
ruby
  Project: openshift
  Tags:    2.0, 2.2, 2.3, latest
wildfly
  Project: openshift
  Tags:    10.0, 10.1, 8.1, 9.0, latest
```